### PR TITLE
Feature/new ui

### PR DESCRIPTION
**Pull request by girishranganathan**

### CODE REVIEW

**Comment by girishranganathan** [comment]
@leecalcote, lets get this merged ASAP before we start the next set of tasks

File: ui/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            <Grid item>
+              <Tooltip title="Alerts • No alters">
+                <IconButton color="inherit">
+                  <NotificationsIcon />
+                </IconButton>
+              </Tooltip>
+            </Grid>
+            <Grid item>
+              <IconButton color="inherit" className={classes.iconButtonAvatar}>
+                <Avatar className={classes.avatar} src="/static/images/avatar/1.jpg" />
```
Comment (leecalcote) [inline ui/components/Header.js]: I was looking earlier today, but couldn't quite figure out where to insert a whitespace (&nbsp;) between the avatar icon and the username. I suspect there's a component for that (whitespace) or CSS to add padding to.

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,50 +1,18 @@
...
-      - dynamo
   fortio:
     image: fortio/fortio
     command: server
     restart: always
   meshery:
     image: layer5/meshery
     environment:
-      - "SAAS_BASE_URL=http://meshery-saas:9876"
+      - "SAAS_BASE_URL=https://meshery.layer5.io"
```
Comment (leecalcote) [inline docker-compose.yaml]: Nice!

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,50 +1,18 @@
...
       - "EVENT=mesheryLocal"
       - "FORTIO_URL=http://fortio:8080/fortio/"
     ports:
-      - "8080:8080"
+      - "9081:8080"
```
Comment (leecalcote) [inline docker-compose.yaml]: Any significance to the port 9081? Just needed to pick one?

File: hooks/build [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker build -t layer5/meshery:latest .
+docker tag layer5/meshery:latest layer5/meshery:$SOURCE_COMMIT
```
Comment (leecalcote) [inline hooks/build]: Where does `$SOURCE_COMMIT` come from? I assume this is reference to the commit ID on GitHub sent via webhook on PR merge...

**Pull request merged**